interfaces/Portalicious/src/app/domains/notification/notification.api.service.ts

```diff
@@ -12,12 +17,37 @@ const BASE_ENDPOINT = (projectId: Signal<number>) => [
   providedIn: 'root',
 })
 export class NotificationApiService extends DomainApiService {
+  private locale = inject<Locale>(LOCALE_ID);
+  private readonly translatableStringService = inject(
+    TranslatableStringService,
+  );
+
   getMessageTemplates(projectId: Signal<number | undefined>) {
-    return this.generateQueryOptions<MessageTemplate[]>({
+    return this.generateQueryOptions<
+      MessageTemplate[],
+      MessageTemplateWithTranslatedLabel[]
+    >({
       path: [
         ...BASE_ENDPOINT(projectId as Signal<number>),
         'message-templates',
       ],
+      processResponse: (response) => {
+        return response
+          .filter(
+            (template) =>
+              template.isSendMessageTemplate &&
+              template.language === (this.locale as string),
```

Comment:
@aberonni the current portal always shows the English template and a program could have some untranslated templates, so for now this should be reverted to just English

```suggestion
              template.language === 'en',
```

Reply:
Checked in with @tijsziere and we can leave it as is: the templates will be loaded in the users' language, under the assumption that translations will be available.